modules/apps/object/object-dynamic-data-mapping-storage/src/main/java/com/liferay/object/dynamic/data/mapping/storage/internal/ObjectDDMStorageAdapter.java

```diff
@@ -49,8 +88,42 @@ public DDMStorageAdapterGetResponse get(
 			DDMStorageAdapterGetRequest ddmStorageAdapterGetRequest)
 		throws StorageException {
 
+		DDMForm ddmForm = ddmStorageAdapterGetRequest.getDDMForm();
+
+		ObjectEntry objectEntry;
+
+		try {
+			objectEntry = _objectEntryManager.getObjectEntry(
+				_getDTOConverterContext(
+					ddmStorageAdapterGetRequest.getPrimaryKey(), null,
+					ddmForm.getDefaultLocale()),
+				ddmStorageAdapterGetRequest.getPrimaryKey());
+		}
+		catch (Exception exception) {
+			throw new StorageException(exception);
+		}
+
+		Map<String, Object> values = objectEntry.getProperties();
```

Comment:
@gabrielwas, shouldn't we name this variable as `properties` once we're filling it through the result of a method named `getProperties`?

Usually, this is a kind of consistency that Brian looks for. What do you think?

Reply:
You're right! Thanks, Renato! 😄